src/desktop/apps/artwork/components/metadata/components/artwork_classification/index.coffee

```diff
@@ -0,0 +1,74 @@
+        </div>
+        <div class='attribution-class-list'>
+          <div class='attribution-class-row'>
+            <div class='attribution-class-name'>Unique</div>
+            <div class='attribution-class-description'>
+              One of a kind piece, created by the artist.
+            </div>
+          </div>
+          <div class='attribution-class-row'>
+            <div class='attribution-class-name'>Limited edition</div>
+            <div class='attribution-class-description'>
+              Original works created in multiple with direct involvement of the artist.
+              Generally, editions are smaller than 150 pieces total.
+            </div>
+          </div>
+          <div class='attribution-class-row'>
+            <div class='attribution-class-name'>Made-to-order</div>
+            <div class='attribution-class-description'>
+              A piece that is made-to-order, taking into account the collector’s preferences.
+            </div>
+          </div>
+          <div class='attribution-class-row'>
+            <div class='attribution-class-name'>Reproduction</div>
+            <div class='attribution-class-description'>
+              Reproduction of an original work authorized by artist’s studio or estate.
+              The artist was not directly involved in production.
+            </div>
+          </div>
+          <div class='attribution-class-row'>
+            <div class='attribution-class-name'>Editioned multiple</div>
+            <div class='attribution-class-description'>
+              Pieces created in larger limited editions, authorized by the artist’s studio or estate.
+              Not produced with direct involvement of the artist.
+            </div>
+          </div>
+          <div class='attribution-class-row'>
+            <div class='attribution-class-name'>Non-editioned multiple</div>
+            <div class='attribution-class-description'>
+              Works made in unlimited or unknown numbers of copies, authorized by the artist’s studio or estate.
+              Not produced with direct involvement of the artist.
+            </div>
+          </div>
+          <div class='attribution-class-row'>
+            <div class='attribution-class-name'>Ephemera</div>
+            <div class='attribution-class-description'>
+              Peripheral artifacts related to the artist.
+              Exhibition materials, memorabilia, photos of the artist, autographs, etc.
+            </div>
+          </div>
+        </div>
+        <div>
+          <button class='attribution-class-close-btn avant-garde-button-black is-block js-modalize-close'>Ok</button>
+        </div>
+      </div>
+    </div>
+  """
+  $el = $(attributionClassDetailsHtml)
+
+  view = render: -> $el: $el
+
+  modal = modalize view,
+    className: 'modalize artwork-classification-modal'
+    dimensions: width: '600px'
+  modal.open()
+  modal
```

Comment:
#minor. I am not too familiar with `modalize` but looks like what this function does is initializing a modal ***and*** opening it. I wonder if it's more modular to make this only responsible for creating a modal, and let the [client](https://github.com/artsy/force/pull/2249/files#diff-71e5964b096c1c552f0a6580a88e8219R8) open it.

Also, it feels unclear that all this `artwork_classification/index.coffee` component does is the modal. Maybe we can rename it to something direct or wrap this in a named export (in case we will have more). Like
```coffeescript
module.exports.giveMeAModal = ->
  # ...
```

Reply:
Seems like it's the pattern here that index creates the modal and opens it right away. I've seen it in several places. And since we not doing anything complicated here I've just followed the pattern.

But I did rename the component to `artwork_classification_modal` to be more explicit about what is going on here.